ex03.exs

```diff
@@ -59,8 +59,14 @@ defmodule Ex03 do
 
   """
 
+  
   def pmap(collection, process_count, function) do
-    « your code here »
+    number =Enum.count collection
```

Comment:
No deduction, but I think lines 64 and 65  would be easier to read if combined:

~~~ elixir
collection |> Enum.count |> div(process_count)
~~~

"Take a collection, count its elements, and divide the result by process count"